frontend/talentsearch/src/js/components/Browse/BrowsePoolsPage.tsx

```diff
@@ -0,0 +1,327 @@
+                    "Title for the current jobs recruiting candidates",
+                })}
+              </Heading>
+              <p data-h2-margin="base(x1, 0)" data-h2-font-weight="base(700)">
+                {intl.formatMessage({
+                  id: "gtaSs1",
+                  defaultMessage:
+                    "This platform allows you to apply to recruitment processes that makes it easy for hiring managers to find you.",
+                  description:
+                    "Description of how the application process works, paragraph one",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  id: "EIHPGF",
+                  defaultMessage:
+                    "Your application to a process will be reviewed by our team and if it's a match, you will be invited to an assessment. Once accepted, managers will be able to contact you about job opportunities based on your skills.",
+                  description:
+                    "Description of how the application process works, paragraph two",
+                })}
+              </p>
+              <div data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)">
+                <ul
+                  data-h2-margin="base(0)"
+                  data-h2-padding="base(0)"
+                  data-h2-list-style="base(none)"
+                >
+                  {filteredPoolAdvertisements.map((poolAdvertisement) => (
+                    <li key={poolAdvertisement.id}>
+                      <PoolCard pool={poolAdvertisement} />
+                    </li>
+                  ))}
+                </ul>
+                <div
+                  data-h2-background-color="base(white) base:dark(black.light)"
+                  data-h2-shadow="base(large)"
+                  data-h2-padding="base(x1) p-tablet(x2)"
+                  data-h2-radius="base(rounded)"
+                >
+                  <div
+                    data-h2-display="p-tablet(flex)"
+                    data-h2-gap="base(x3)"
+                    data-h2-align-items="base(center)"
+                  >
+                    <div>
+                      <Heading
+                        level="h3"
+                        size="h6"
+                        data-h2-margin="base(0, 0, x1, 0)"
+                      >
+                        {intl.formatMessage({
+                          defaultMessage: "More opportunities are coming soon!",
+                          id: "g+JcDC",
+                          description:
+                            "Heading for message about upcoming opportunities",
+                        })}
+                      </Heading>
+                      <p>
+                        {intl.formatMessage({
+                          defaultMessage:
+                            "We're posting new opportunities all the time. By starting your profile now, you'll be able to submit applications lightning fast when the time comes.",
```

Comment:
What if I'm already logged in and have a profile?  This message might be confusing.
![image](https://user-images.githubusercontent.com/8978655/196708249-ac013900-9c2a-490f-9f06-0d76a57d925e.png)

Reply:
"Update my profile"

Good catch!

Reply:
Thanks, updated with [update profile button text while logged in](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4293/commits/81f52872604ca09c2d430c9651b01af55c6e12bc)

Reply:
This fixes the button.  Do we need to update the paragraph, too?  "By keeping your profile up to date, you'll be able to..."?